documents/src/pages/elements/list.md

```diff
@@ -68,7 +68,7 @@ if (list) {
 
 ## Data property interface
 
-The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) interface for its data items.
+The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) interface for its data items. Each item defaults to type `ItemData`, it can be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
```suggestion
The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) interface for its data items. Each item defaults to type `ItemData`, otherwise it can be any interface that extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Reply:
Updated by following P'Ole suggestions of comboBox pattern.